src/Softeq.NetKit.Chat.Web/appsettings.Development.json

```diff
@@ -1,5 +1,48 @@
 ﻿{
```

Comment:
never commit your local config file

Reply:
@Gredja  it's better to remove this PR and revert changes to remove them from history